kinetic/distribution.yaml

```diff
@@ -8076,6 +8076,22 @@ repositories:
       url: https://github.com/ros-planning/random_numbers.git
       version: master
     status: maintained
+  raspi_temperature:
+    doc:
+      type: git
+      url: https://github.com/bberrevoets/raspi_temperature.git
+      version: 0.1.1
```

Comment:
This doesn't seem to be a valid branch name. You probably mean `master`. Same for the source entry.